ProjectSettings/Packages/com.unity.probuilder/Settings.json

```diff
@@ -123,6 +123,16 @@
                 "key": "smoothing.previewDither",
                 "value": "{\"m_Value\":false}"
             },
+            {
```

Comment:
Probably shouldn't add this file (since it's one of the problem files)?
https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request